web/src/routes/+error.svelte

```diff
@@ -54,6 +55,11 @@
                 <label for="stacktrace">Stacktrace</label>
                 <pre id="stacktrace" class="text-xs">{$page.error?.stack || 'No stack'}</pre>
               {/if}
+              {#if $page.error?.code}
+                <a href="/">
+                  <Button size="sm">Go back</Button>
+                </a>
+              {/if}
```

Comment:
should we not show this button all the time? I don't think it needs to be limited to when there is an error code.

Reply:
I think we should keep it only for 404 errors, the button is at best useless for other errors at worst misleading